package.json

```diff
@@ -94,11 +94,14 @@
     "webpack": "^1.13.2"
   },
   "dependencies": {
+    "aphrodite": "^0.5.0",
```

Comment:
Yes, you need to do this in a way that doesn't couple this project to a particular interface. @adamrneary is doing something similar with Rheostat right now, so it might be a good idea to sync up about how to best accomplish this. Relevant PR: https://github.com/airbnb/rheostat/pull/38